shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/LeafFormat.kt

```diff
@@ -34,8 +34,14 @@ sealed class LeafFormat {
         override fun noPredictionsStatus() = (format as? UpcomingFormat.NoTrips)?.noTripsFormat
     }
 
-    /** A [RouteCardData.Leaf] which has multiple destinations within its direction. */
-    data class Branched(val branches: List<Branch>) : LeafFormat() {
+    /**
+     * A [RouteCardData.Leaf] which has multiple destinations within its direction. The secondary
+     * alert will be displayed once for the entire direction
+     */
+    data class Branched(
+        val branches: List<Branch>,
```

Comment:
not directly related to this PR, but I was thinking about renaming `Branch` here something like `rows` since it isn't quite the typical meaning of branch. Ex: I think of "Braintree" as one branch, but the list of branches here could include Braintree three times.

Reply:
Yeah, I guess it’s weird that the list doesn’t contain the distinct branches of the route. I don’t know that just `Row` as the name of the class would make sense, though; maybe `BranchRow`?